src/data/code-samples/en/input-instructions/swift-ui.md

```diff
@@ -1,32 +1,24 @@
 # Input instructions - SwiftUI
 
-In SwiftUI, we recommend using a [`Text`](https://developer.apple.com/documentation/swiftui/text) or [`Label`](https://developer.apple.com/documentation/swiftui/label) views to show input instructions. The input instructions message should also be posted to assistive technologies by using an [`accessibility announcement`](../Techniques/accessibility-announcement.md).
-
-You could also use a third party library to displaying instructions. Unfortunately, accessibility is often not considered in the implementations.
+In SwiftUI, we recommend using [`Text`](https://developer.apple.com/documentation/swiftui/text) or [`Label`](https://developer.apple.com/documentation/swiftui/label) views to show input instructions. The input instructions message should also be posted to assistive technologies by using an [`accessibility announcement`](../Techniques/accessibility-announcement.md).
```

Comment:
accessibility-announcement URL should use full appt.org link (cannot use relative links, because the code samples folder itself is not published)